src/runtime_src/core/tools/xbmgmt2/SubCmdFlash.cpp

```diff
@@ -49,7 +49,7 @@ static void
 scan_devices(bool verbose, bool json)
 {
   json = json;
-  auto total = xrt_core::get_total_devices().first;
+  auto total = xrt_core::get_total_devices(false).first;

```

Comment:
Hmm....
I looks like get_total_devices() returns different information if the user driver is being examined as opposed to the mgmt driver (hence the changes).  By chance do you know why for I would expect the same results.

Reply:
When the device is ready to use, you'll have both xocl and xclmgmt devices. I had to add this so that xbmgmt flash --scan displays the correct information when the golden image is loaded. In that case, the user driver is not loaded but the mgmt driver is.